types/coingecko-api/index.d.ts

```diff
@@ -1,8 +1,54 @@
 // Type definitions for coingecko-api 1.0
 // Project: https://github.com/miscavage/CoinGecko-API#readme
-// Definitions by: Jan Klimo <https://github.com/janklimo>
+// Definitions by: Jan Klimo <https://github.com/janklimo>, Artem Ilinykh <https://github.com/singlesly>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+type Locale = "en"|"de"|"es"|"fr"|"it"|"pl"|"ro"|"hu"|"nl"|"pt"|"sv"|"vi"|"tr"|"ru"|"ja"|"zh"|'zh-tw'|"ko"|"ar"|"th"|"id";
+
+type Localization = Record<Locale, string>;
+
+type Currency = "usd";
```

Comment:
Since we've provided an exhaustive list of locales, maybe do the same here as well? Here's the response:

```
  "market_data": {
    "current_price": {
      "aed": 27157.848106085992,
      "ars": 442015.9505785273,
      "aud": 10587.436593658975,
      "bch": 34.759038199137784,
      "bdt": 627946.8616670666,
      "bhd": 2787.8669839044596,
      "bmd": 7394.029285530991,
      "bnb": 524.3308043645355,
      "brl": 29919.19270694328,
      "btc": 1,
      "cad": 9666.155332740376,
      "chf": 7200.106079459367,
      "clp": 5537379.585158725,
      "cny": 51727.15007571764,
      "czk": 168212.6874399724,
      "dkk": 49386.91762475931,
      "eos": 2751.411213746318,
      "eth": 55.049354137367715,
      "eur": 6611.740987121814,
      "gbp": 5646.450463895742,
      "hkd": 57584.25643395816,
      "huf": 2190016.4832800427,
      "idr": 103128363.0518223,
      "ils": 25571.008087088936,
      "inr": 528167.5419511257,
      "jpy": 809176.6859060123,
      "krw": 8582630.059247939,
      "kwd": 2243.7181866943747,
      "lkr": 1341586.773980592,
      "ltc": 171.42539765556515,
      "mmk": 10987005.59595389,
      "mxn": 139252.53730150897,
      "myr": 30518.855876029204,
      "ngn": 2683218.88814876,
      "nok": 65122.173529385655,
      "nzd": 11025.007852747447,
      "php": 375518.1474766858,
      "pkr": 1145134.2479470957,
      "pln": 28170.807936115947,
      "rub": 458876.82174337836,
      "sar": 27737.591760276704,
      "sek": 68979.12102559712,
      "sgd": 9998.265552129287,
      "thb": 222875.96957482884,
      "try": 43991.77542822014,
      "twd": 222554.11487405872,
      "uah": 175158.3797597755,
      "usd": 7394.029285530991,
      "vef": 1837324903.8224971,
      "vnd": 170786800.94144997,
      "xag": 414.8131024102093,
      "xau": 4.8837563430932125,
      "xdr": 5357.380888977906,
      "xlm": 159650.9365970607,
      "xrp": 37660.063649842705,
      "zar": 103938.97877110212,
      "bits": 1000000,
      "link": 3898.0341381720714,
      "sats": 100000000
    },
```

Reply:
done!